unblob/handlers/compression/lzh.py

```diff
@@ -0,0 +1,106 @@
+import io
+from typing import List, Optional
+
+from structlog import get_logger
+
+from ...file_utils import Endian
+from ...models import StructHandler, ValidChunk
+
+logger = get_logger()
+
+PADDING_LEN = 2
```

Comment:
This is not used (sorry I did not catch earlier)

Reply:
Now it's used.